modules/openapi-generator/src/main/java/org/openapitools/codegen/DefaultGenerator.java

```diff
@@ -432,17 +432,17 @@ void generateModels(List<File> files, List<Object> allModels, List<String> unuse
         for (String name : modelKeys) {
             try {
                 //don't generate models that have an import mapping
-                if (config.importMapping().containsKey(name)) {
-                    LOGGER.debug("Model {} not imported due to import mapping", name);
-
-                    for (String templateName : config.modelTemplateFiles().keySet()) {
-                        // HACK: Because this returns early, could lead to some invalid model reporting.
-                        String filename = config.modelFilename(templateName, name);
-                        Path path = java.nio.file.Paths.get(filename);
-                        this.templateProcessor.skip(path,"Skipped prior to model processing due to import mapping conflict (either by user or by generator)." );
-                    }
-                    continue;
-                }
+//                if (config.importMapping().containsKey(name)) {
+//                    LOGGER.debug("Model {} not imported due to import mapping", name);
+//
+//                    for (String templateName : config.modelTemplateFiles().keySet()) {
+//                        // HACK: Because this returns early, could lead to some invalid model reporting.
+//                        String filename = config.modelFilename(templateName, name);
+//                        Path path = java.nio.file.Paths.get(filename);
+//                        this.templateProcessor.skip(path,"Skipped prior to model processing due to import mapping conflict (either by user or by generator)." );
+//                    }
+//                    continue;
```

Comment:
Question to maintainers: does it make sense to do this stuff configurable? Like introduce property this will turn this feature on/off

Reply:
I'm not sure to understand why you want to remove this part.

Reply:
Technically there can be a case, when client of yaml/json specification would like you you his own DTO instead of generated by openapi specification

Regarding solution: I would prefer to introduce configuration parameter so that client can configure this behaviour

Reply:
It's the case of this block, skip models override in the importMapping.
It's also possible to just skip all model creation and just use the right `modelPackage` if his own DTO has the same name like this :
```
<modelPackage>my.package.of.dto</modelPackage>
<generateModels>false</generateModels>
```

Reply:
@Zomzog idea in original issue was to allow custom override for some of models and not all

Reply:
Yep and model per model it's the `importMapping` option. But I think I missing something. Could you give me an example where this part creates issues? The related example doesn't work but I think for another reason.

Reply:
@Zomzog https://github.com/OpenAPITools/openapi-generator/blob/326d890c4a1882953f88dabe828244100b1bb9ed/modules/openapi-generator/src/test/resources/3_0/issue_8548.yaml

example from `samples/server/petstore/spring-mvc-custom-import-with-override` contains custom import mapping for object `CustomObject` and therefore this object won't be generated(https://github.com/OpenAPITools/openapi-generator/pull/8661/files#diff-21792377bac6a73df584a020e525e172900373ab1ff04f3112fe08df88183878R24)

example from `samples/server/petstore/spring-mvc-custom-import-without-override` doesn't have override and therefore `CustomObject` will be generated by generator

Not sure why but this commented peace of code doesn't allow partial model override